### PR TITLE
Add feature flag "conferenceCalling"

### PR DESCRIPTION
**Pull request by smatting**
https://wearezeta.atlassian.net/browse/SQSERVICES-681

This PR also:
- refactor some of the feature endpoint implementation
- adds to test that all simple flags should emit a `feature-config.update` event

## Checklist

 - [x] The **PR Title** explains the impact of the change.
 - [x] The **PR description** provides context as to why the change should occur and what the code contributes to that effect. This could also be a link to a JIRA ticket or a Github issue, if there is one.
 - [x] If end-points have been added or changed: the **endpoint / config-flag checklist** (see Wire-employee only backend [wiki page](https://github.com/zinfra/backend-wiki/wiki/Checklists)) has been followed.
 - [x] If a schema migration has been added, I ran **`make git-add-cassandra-schema`** to update the cassandra schema documentation.
 - [x] Section *Unreleased* of **CHANGELOG.md** contains the following bits of information:
   - [x] A line with the title and number of the PR in one or more suitable sub-sections.
   - [x] If /a: measures to be taken by instance operators.
   - [x] If /a: list of cassandra migrations.
   - [x] If public end-points have been changed or added: does nginz need upgrade?
   - [x] If internal end-points have been added or changed: which services have to be deployed in a specific order?

### CODE REVIEW

**Review by jschaul — COMMENTED** [review]
Could you update the docs/config-options.md and explain what it means for this to be enabled or disabled, and what the default is?

**Comment by smatting** [comment]
> Could you update the docs/config-options.md and explain what it means for this to be enabled or disabled, and what the default is?

Done

File: services/federator/federator.cabal [hunk 1]
```diff
@@ -17,8 +17,6 @@ license:        AGPL-3
 build-type:     Simple
 extra-source-files:
     test/resources/integration-ca.pem
-    test/resources/integration-leaf-key.pem
-    test/resources/integration-leaf.pem
```
Comment (fisx) [inline services/federator/federator.cabal:21]: this seems unrelated?

**Review by fisx — APPROVED** [review]
LGTM!

No point in changing this now, but I would have preferred 2-3 smaller PRs (at least the unrelated tests should have been trivial to do separately).
